### ISSUE TITLE
Query parameters support

### ISSUE THREAD

**Issue by danicc097** [UniqueVision/plpgsql-lsp#2]
### Is your enhancement related to a problem? Please describe.

Error raised when using query parameters

### Describe the solution you would like

It would be great to have support for parameterized queries or an option to ignore them:

```sql
select * from users where email = any(@emails::text[]) -- Syntax error: error: column "emails" does not exist

select * from users where email = any(:emails) --  Syntax error: error: syntax error at or near ":"
```

### Describe alternatives you have considered

Right now they block the analysis of the file so the only workaround is to replace them with an actual value

### Additional context

The use case is for backend query building, e.g.
```sql
-- name: getUsersByEmails
select * from users where email = any(@emails::text[]) 

-- name: ...
...
```
```python
database.execute(query=queries["getUsersByEmails"], values={"emails": ["test@mail.com", "test2@mail.com"]})
```

**Comment by yas7010uv** [comment]
Keyword parameters (@name, :name, etc.) are not interpreted by the [pg library](https://node-postgres.com/). 
Also, in the SQL standard support positional parameters ($1, $2,...) only.

I'll consider about positional parameters support.

**Comment by yas7010uv** [comment]
@danicc097 

Today, we have released [v2.2.0](https://github.com/UniqueVision/plpgsql-lsp/tree/v2.2.0).
By adding a comment at the top of the file, you can validate query using positional/keyword parameter.

See [README.md](https://github.com/UniqueVision/plpgsql-lsp/tree/v2.2.0#experimental-feature-query-parameters) for details.
